syntax = "proto3";

package order;



// import "google/protobuf/timestamp.proto";

message Status {
  enum Value {
    PENDING = 0;
    COMPLETED = 1;
    IN_PROGRESS = 2;
  }
  Value value = 1;
}

message Order {
  string id = 1;
  string user_id = 2;
  string product_id = 3;
  int32 quantity = 4;
  int32 total_amount = 5;
  string status = 6;
  string created_at = 7;
  // google.protobuf.Timestamp created_at = 7;
}